### DISCUSSION TITLE
Why do folders appear that I have already deleted?

### DISCUSSION THREAD

**Discussion by Officepoint** [rejetto/hfs#456] [General]
Hi!
Why do folders appear that I have already deleted?
As you can see in the Admin Panel I only have three folders, but when I access the server page it shows all the others that have already been deleted.
I've tried Google Chrome, Edge, and incognito mode but they still appear.

Thanks! 

![Hfs 1](https://github.com/rejetto/hfs/assets/94564442/5fb08666-6a5f-4cbf-9309-6ff2dc38f631)
![Hfs 2](https://github.com/rejetto/hfs/assets/94564442/91a72f48-fd49-4574-8a88-9851645adaa3)

**Comment by SanokKule** [comment]
Those folders are from the folder on the disk that you selected here
![Screenshot_2024-01-22-09-11-43-455_com chrome beta](https://github.com/rejetto/hfs/assets/24497821/e1065a2e-d9f3-4d40-a133-6f25837a36f2)

**Comment by rejetto** [comment]
so, i guess you were just overlooking that setting.
Please let me know if you think there is something that could be displayed (or phrased) differently to avoid any confusion.

**Reply by Officepoint** [reply]
Yes, that's correct, somehow I overlooked that setting. From my point of view it's a bit hard to understand the permissions system. I sent an email to you with an attempt to explain how I would see the permissions system.
Thanks for the explanation! 
All the best!

**Comment by rejetto** [comment]
ok sure, thanks for it, but the suggestion in your email has nothing to do with the setting you overlooked.
i was asking if there was a suggestion for this specifically.
